src/services/export.js

```diff
@@ -297,14 +317,66 @@ const scrape = async ({ url, format }) => {
     });
 
     Logger.debug('visiting page');
+
+    let auth = AUTH_TYPE_ANONYMOUS;
+    // this endpoint will return a 401 so we catch to find the auth type
+    try {
+      const loginTypeResponse = await request({
+        uri: loginTypeUrl,
+        json: true,
+      });
+      ({ body: { auth } = {} } = loginTypeResponse);
+    } catch (err) {
+      ({ error: { auth } = {} } = err);
+    }
+    console.log(auth);
+
     await page.goto(url, {
       waitUntil: 'networkidle0',
       // one minute timeout
-      timeout: 60000,
+      timeout: TIMEOUT,
     });
 
+    switch (auth) {
+      case AUTH_TYPE_USERNAME:
+        // TODO throw error if no username
+        await page.type('#username', username);
+        await Promise.all([
+          page.waitForNavigation({
+            timeout: TIMEOUT,
+            waitUntil: 'networkidle0',
+          }),
+          page.click('.submit'),
+        ]);
+        break;
+
+      case AUTH_TYPE_PASSWORD:
+        // TODO throw error if no username
+        await page.type('#username', username);
+        await page.type('#password', password);
+        await Promise.all([
+          page.waitForNavigation({
+            timeout: TIMEOUT,
+            waitUntil: 'networkidle0',
+          }),
+          page.click('.submit'),
+        ]);
+
```

Comment:
This empty line in line 364 is not needed. Especially because in the case block above, you don't have an empty line before the `break`.